.sbtopts

```diff
@@ -1,5 +1,3 @@
 -J-Xms512M
 -J-Xmx4096M
--J-XX:-UseConcMarkSweepGC
--J-XX:+UseG1GC
```

Comment:
FYI, I had to remove these two lines because of the following errors:

```
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
Error occurred during initialization of VM
Multiple garbage collectors selected
```